app/models/holiday_import.rb

```diff
@@ -95,14 +95,14 @@ def populate_from_ical_event(cal_event)
   end
 
   def populate_from_suggestions
-    begin
+    
       holiday_info = Holidays.between(start_date, end_date, @country_code.to_sym, :observed)
       holiday_info.each do |holiday_info_hash|
         holidays << Holiday.new(description: holiday_info_hash[:name],
                                 day: holiday_info_hash[:date])
       end
     rescue Holidays::InvalidRegion
       []
-    end
+    
```

Comment:
Where it removes these redundant begin/end blocks, the contents should also be shifted left to get back to two-spacing.

Simple example:

```rb
# BEFORE
def foo
  begin
    bar
  rescue SomeError
    # do something
  end
end

# AFTER AUTOCORRECT
def foo
    bar
  rescue SomeError
    # do something
end

# WHAT WE WANT
def foo
  bar
rescue SomeError
  # do something
end
```

Reply:
This has been changed and squashed into that commit